### PR TITLE
Add endpoints

### PR DESCRIPTION
**Pull request by aneallaryea100**
# Blog app: add API endpoints

- Create an API endpoint to list all posts for a user.

Screenshot (112)

- Create an API endpoint to list all comments for a user's post.

Screenshot (113)

- Create an API endpoint to add a comment to a post. Remember that the owner of the comment is the user that makes it; it should **not** be set by passing a parameter.

Screenshot (114)

- Add authentication to your API endpoints.
- Your API endpoints should receive JSON and respond JSON as well.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,10 +6,22 @@
...
+        end
+      end
+    end
+  end
+  
   resources :users, only: [:index, :show] do
     resources :posts, only: [ :index, :new, :create, :show, :destroy] 
     resources :comments, only: [ :create, :destroy]
     resources :likes, only: [ :create ]
   end
```
Comment (DeliceLydia) [inline config/routes.rb:24]: - [ ] Please add an API endpoint for authentication, both registration, and login
for example:
* Login endpoint - **POST :** `/login`
* Registration endpoint - **POST :** `/register` or `/signup`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: config/routes.rb (hunk 1 above)
Comment (aneallaryea100) [reply config/routes.rb:24]: @DeliceLydia 
The login and registration endpoints is working very well on my side.
You can log out of the app to check the authentication. Everything is working perfectly but I do not understand why is not working on your side.

Screenshot (115)

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @aneallaryea100 and @41hulk 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**